app/models/publication.rb

```diff
@@ -0,0 +1,9 @@
+class Publication < ApplicationRecord
+    has_many :papers
+    validates :name, presence: true
+    before_save :downcase_fields
+
+    def downcase_fields
+        self.name.downcase!
```

Comment:
Also, \<pedantic>this method name isn't exactly accurate, as it just downcases one field, right?\</pedantic>